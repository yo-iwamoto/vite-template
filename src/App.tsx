export function App() {
  return (
    <h1 className='flex min-h-screen items-center justify-center text-center text-5xl font-bold italic'>
      Hello Vite⚡️
    </h1>
  );
}
