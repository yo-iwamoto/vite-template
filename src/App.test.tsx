import { App } from './App';
import { render } from '@testing-library/react';

describe('App', () => {
  it('見出しが表示されること', () => {
    const { getByRole } = render(<App />);

    expect(
      getByRole('heading', { name: 'Hello Vite⚡️', level: 1 }),
    ).toBeInTheDocument();
  });

  it('DOM snapshot matches', () => {
    const { asFragment } = render(<App />);

    expect(asFragment()).toMatchSnapshot();
  });
});
