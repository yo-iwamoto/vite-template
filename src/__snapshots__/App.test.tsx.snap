// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`App > DOM snapshot matches 1`] = `
<DocumentFragment>
  <h1
    class="flex min-h-screen items-center justify-center text-center text-5xl font-bold italic"
  >
    Hello Vite⚡️
  </h1>
</DocumentFragment>
`;
